### DISCUSSION TITLE
Update sample code for Home Assistant integration

### DISCUSSION THREAD

**Discussion by alexrusso51** [teslamate-org/teslamate#2684] [General]
Home Assistant is deprecating the configuration of templates and MQTT entities directly under the respective platform keys. All MQTT entities will need to be configured under the MQTT configuration key instead. 

The sample code in the documentation uses the soon-to-be deprecated formating. Can this please be updated?

**Comment by tobiasehlert** [comment]
PR #2704 might be what you are asking for @alexrusso51 :)

**Reply by alexrusso51** [reply]
Great! Glad someone is on it. I already updated all my MQTT entities if anyone wants the code to use a sample.
